client/src/context/useSocketContext.tsx

```diff
@@ -22,7 +24,13 @@ export const SocketProvider: FunctionComponent = ({ children }): JSX.Element =>
       }),
     );
   }, []);
+  useEffect(() => {
+    socket && loggedInUser && socket.emit('addUser', loggedInUser.email);
 
+    if (loggedInUser === null) {
```

Comment:
You can just test for Falsiness of `loggedInUser` `if (!loggedInUser)`

Reply:
Fixed